packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -0,0 +1,24 @@
+import { useStore } from '@deriv/stores';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+const useTotalAccountBalance = (accounts: DetailsOfEachMT5Loginid[]) => {
+    const { exchange_rates } = useStore();
+    const currency = accounts[0]?.currency;
```

Comment:
```suggestion
    const currency = accounts[0]?.currency || 'USD';
```
Not sure tho 👀

Reply:
this is valid. but I think we have to change all hard-coded  `USD`s with `default_currency` which comes from `client` store. but I'm not if it's a good idea to fetch `client` in our hooks just for this value :thinking: